aptos-move/block-executor/src/proptest_types/baseline.rs

```diff
@@ -362,12 +362,16 @@ impl<K: Debug + Hash + Clone + Eq> BaselineOutput<K> {
                     assert_none!(output.materialized_delta_writes.get());
                 });
             },
-            Err(BlockExecutionError::FatalVMError((idx, executor_idx))) => {
-                assert_eq!(*idx, *executor_idx as usize);
+            Err(BlockExecutionError::FatalVMError(idx)) => {
                 assert_matches!(&self.status, BaselineStatus::Aborted);
                 assert_eq!(*idx, self.read_values.len());
                 assert_eq!(*idx, self.resolved_deltas.len());
             },
+            Err(BlockExecutionError::FallbackToSequential(PanicOr::Or(
```

Comment:
Why we have an extra match case here, when there is `Err(BlockExecutionError::FallbackToSequential(e))` below?

Reply:
@georgemitenkov Looks like we have it b/c we only expect the PanicOr variant of e.
@gelash I'd re-do this match (to have one enum variant, and the redo on the matched e - or use match!). This is for performance reasons (to allow for a jump table).

Reply:
because this one is now okay, but the rest is unreachable. I can add explicit variants (it was in a test)